### DISCUSSION TITLE
SQL server not found or unaccessable

### DISCUSSION THREAD

**Discussion by chengdemin** [dotnet/aspnetcore#48631] [Q&A] [answered]
i'm learning this project from MS. At the step "add a model", i ran into a problem. when i ran the app and add "/Movies" into the url.
visual studio report the error "provider: SNI_PN11, error: 50 - Local Database Runtime error occurred". I don't know how to fix it. Thanks in advance , could somebody help me?

**Comment by gfoidl** [comment, marked as answer]
> i'm learning this project from MS. At the step "add a model"

Can you please provide a link to the learning project you were following along?
Also provide information about the basic setup you have. Is the connection string to the database correct, are all the pre-requisites installed and available?

Side note: please keep in mind that this repo here is for development of ASP.NET Core, so you may get better advices / help when asking questions like this one on https://stackoverflow.com/ and similar sites.

**Reply by chengdemin** [reply]
Sorry that i didn't make it clear. here's the link of the project i am following  "https://learn.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/model?source=recommendations&view=aspnetcore-7.0&tabs=visual-studio ". 
it seems that i forgot to use [Entity Framework Core](https://learn.microsoft.com/en-us/ef/core) to deal with the database.
thank you for answering my question. It's so kind of you. I'll take your advice to ask questions on stackoverflow.
By the way. SIr, i'm wondering if you could give me some advice about learning C# in developing website. 
may you have a good day

**Reply by gfoidl** [reply]
I'm glad you could solve that problem.

> give me some advice about learning C# in developing website.

In regards to learning it depends on you learning style. Some people prefer the docs, others blogs, and others watching videos.
For docs a good starting point is [Tutorial: Get started with ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-7.0&tabs=windows).
So depending on your learning style try searching for "ASP.NET Core" maybe together with "tutorial", "basics", etc. I'm pretty sure you'll find enough information to get started. Good luck and enjoy your journey with ASP.NET Core.

**Reply by chengdemin** [reply]
I'm really appreciate with your advice sir. I started this project with google "how to build a website with C#". So actually i have no ideal on where to start. 
About the learning material, both docs and videos are suitablt for me. Now i'll start my learning from the docs.
Wish you good luck Sir
and with my best regards.
